File: code/backend/java/core/src/main/java/games/paths/core/model/dev/CleanupResult.java
package games.paths.core.model.dev;

/**
 * CleanupResult - immutable summary of a dev-only test-data cleanup operation.
 *
 * @param deletedGuests  number of guest users removed
 * @param deletedMatches number of matches removed
 */
public record CleanupResult(int deletedGuests, int deletedMatches) {
}

File: code/backend/php/src/Core/Domain/Dev/CleanupResult.php
<?php

namespace Games\Paths\Core\Domain\Dev;

/**
 * Immutable summary of a dev-only test-data cleanup operation.
 */
final class CleanupResult
{
    public function __construct(
        public readonly int $deletedGuests,
        public readonly int $deletedMatches
    ) {
    }
}
